--- lal-7.5.0/Makefile.am.orig	2024-06-03 06:11:27.718254594 +0200
+++ lal-7.5.0/Makefile.am	2024-06-03 06:18:51.389233063 +0200
@@ -26,11 +26,6 @@ user_environment = \
 	trailingcolon MANPATH; \
 	set $(PACKAGE)_PREFIX $(prefix); \
 	set $(PACKAGE)_DATADIR $(pkgdatadir); \
-	prepend MANPATH $(mandir); \
-	prepend OCTAVE_PATH $(octexecdir); \
-	prepend PATH $(bindir); \
-	prepend PKG_CONFIG_PATH $(libdir)/pkgconfig; \
-	prepend PYTHONPATH $(pyexecdir) $(pythondir); \
 	$(END_OF_LIST)
 
 pkgconfigdir = $(libdir)/pkgconfig
